docs/helm-charts.md

```diff
@@ -1,17 +1,21 @@
 # Helm Charts
 
-k0s supports deploying applications with Helm charts in two ways:
+Helm charts is one of two methods you can use to run k0s with your preferred extentions (the other being [Manifest Deployer](manifests.md), which is included with k0s).
 
-- Helm command can be used in runtime to install applications like described in the [Helm Quickstart Guide](https://helm.sh/docs/intro/quickstart/).
-- Helm charts can be inserted directly into the k0s configuration file (k0s.yaml). This way a separate install of `helm` tool is not needed and the charts are automatically deployed at the k0s bootstrap phase.
+k0s supports two methods for deploying applications using Helm charts:
+
+- Use Helm command in runtime to install applications. Refer to the [Helm Quickstart Guide](https://helm.sh/docs/intro/quickstart/) for more information.
+- Insert Helm charts directly into the k0s configuration file, ``k0s.yaml``. This method does not require a separate install of `helm` tool and the charts automatically deploy at the k0s bootstrap phase.
 
 ### Helm charts in k0s configuration
 
-When Helm charts are added into k0s configuration file, you'll get a declarative way to configure the cluster. k0s controller manages the setup of the defined extension Helm charts as part of the cluster bootstrap process.
+Adding Helm charts into the k0s configuration file gives you a declarative way in which to configure the cluster. k0s controller manages the setup of the defined extension Helm charts as part of the cluster bootstrap process.
```

Comment:
"manages the setup of the defined extension Helm charts" sounds wrong. Maybe something like "manages the setup of Helm charts that have been defined as extensions in the k0s configuration file" 🤔 ?

Reply:
Integrated the suggestion.